README.md

```diff
@@ -173,6 +173,12 @@ A good place to start is with an issue that has the label [good first issue](htt
 4. Push to the Branch (`git push origin f/AmazingFeature`)
 5. Open a Pull Request
 
+### Adding new API
```

Comment:
I know that we currently lack documentation regarding how to contribute and how to add new APIs. But it doesn't belong into the main readme file. Please create a new issue with the goal to setup a developer documentation.

Reply:
cleaned